processing/context_shipper_amqp_test.go

```diff
@@ -113,7 +115,20 @@ func TestContextShipperAMQPBrokenJSON(t *testing.T) {
 	}
 
 	close(inChan)
-	if entries[0].Message != `could not marshal event JSON: {""value":1}` {
-		t.Fatalf("wrong error message: %v", entries[0].Message)
+	found := false
+
+	for _, entry := range entries {
+		if entry.Message == `could not marshal event JSON: {""value":1}` {
+			found = true
+			break
+		}
+	}
+
+	if !found {
+		var entryStrings bytes.Buffer
+		for i, entry := range entries {
+			entryStrings.WriteString(fmt.Sprintf("(%d: %s)", i, entry.Message))
```

Comment:
This is code to debug a failing test, so it would even be better to use `%03d: %s\n` as format, making it even easier to find out where it is (or where it is not).